<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simpleShapeCircle</class>
 <widget class="QDialog" name="simpleShapeCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="xLabelObject">
        <property name="text">
         <string>xCenter</string>
        </property>
        <property name="buddy">
         <cstring>xCenterLineEditObject</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yLabelObject">
        <property name="text">
         <string>yCenter</string>
        </property>
        <property name="buddy">
         <cstring>yCenterLineEditObject</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="radiusLabelObject">
        <property name="text">
         <string>radius</string>
        </property>
        <property name="buddy">
         <cstring>radiusLineEditObject</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="xCenterLineEditObject"/>
      </item>
      <item>
       <widget class="QLineEdit" name="yCenterLineEditObject"/>
      </item>
      <item>
       <widget class="QLineEdit" name="radiusLineEditObject"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>xCenterLineEditObject</tabstop>
  <tabstop>yCenterLineEditObject</tabstop>
  <tabstop>radiusLineEditObject</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>simpleShapeCircle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>simpleShapeCircle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
